docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -538,6 +541,64 @@ The default value is `fail`, which causes the migrate procedure to throw an
 exception if subdirectories are found. Set the value to `true` to migrate
 nested directories, or `false` to ignore them.
 
+(iceberg-add-files)=
+#### Add files
+
+The connector can add files from tables or locations if
+`iceberg.add-files-procedure.enabled` is set to `true` for the catalog.
+
+Use the procedure `system.add_files` to copy files from the Hive table or specified locations. 
+The data files must be the Parquet, ORC, or Avro file format.
+
+The procedure must be called for a specific catalog `example` with the
+relevant schema and table names supplied with the required parameters
+`target_schema_name` and `target_table_name`:
+
+```sql
+CALL example.system.add_files(
+    target_schema_name => 'testdb',
+    target_table_name => 'iceberg_customer_orders',
+    source_schema_name => 'testdb',
+    source_table_name => 'hive_customer_orders')
+```
+
+You can provide a `partition_filter` argument to add files from specified partitions:
+
+```sql
+CALL example.system.add_files(
+    target_schema_name => 'testdb',
+    target_table_name => 'iceberg_customer_orders',
+    source_schema_name => 'testdb',
+    source_table_name => 'hive_customer_orders',
+    partition_filter => ARRAY['region=ASIA/country=JAPAN'])
+```
+
+In addition, you can provide a `recursive_directory` argument to migrate a
```

Comment:
*thought*: why not just 'recursive'? 
Never mind, I see there is a precedence with `example.system.migrate`